tests/vdaf.rs

```diff
@@ -4,34 +4,46 @@ use test_support::{assert_eq, test, *};
 pub fn histogram_representations() {
```

Comment:
Why have the DAP-04 tests changed? It seems like these should be new test cases, since we want to ensure continued support of all of the previous DAP-04 test cases